README.md

```diff
@@ -70,52 +70,6 @@ functionality of the core spec or add fields for specific domains.
 document, a [list of implementations](implementations.md), [best practices](best-practices.md)
 and a discussion of the collaboration [principles](principles.md) and specification approach.
 
```

Comment:
Somewhere in this section should be a prominent link to the overview.

Reply:
Good call.

Reply:
Decided to make it even more prominent, before the spec directories.